scraper.py

```diff
@@ -92,6 +96,11 @@ def get_slack_message(elections):
             message += " :womble: required"
         if 'locked' in election and election['locked']:
             message += " :lock:"
+        if election['sopn_published'] < str(datetime.date.today()):
+            message += " (SoPN should be published)"
+        else:
+            message += " (SoPN due %s)" % format_date(election['sopn_published'])
```

Comment:
If `election['sopn_published']` is None (for reasons), can we just append nothing here.